doc/source/tutorial/optimize.rst

```diff
@@ -277,6 +275,92 @@ Rosenbrock function using :func:`minimize` follows:
     >>> res.x
     array([1., 1., 1., 1., 1.])
 
+Trust-Region Newton-Conjugate-Gradient Algorithm (``method='trust-ncg'``)
```

Comment:
Are you sure that 'trust-ncg' is a trust-region method? I checked 'tnc' folder and it looks to me that it's a line-search method. Looks like it is just a more thoroughly implemented "Newton-CG". Can you please recheck?